app/typeclasses/confirmation/ResponsiblePeopleRows.scala

```diff
@@ -52,25 +53,28 @@ trait ResponsiblePeopleRows[A] extends FeeCalculations {
                                                         ): Seq[BreakdownRow] = {
 
     val (notPassedFP, notPassedApprovalCheck) = (value.addedResponsiblePeopleFitAndProper, value.addedResponsiblePeopleApprovalCheck)
+    val msbOrTcsp = (activities.contains(MSB) || activities.contains(TrustAndCompanyServices))
+    val bpsTdi = (activities.contains(BPS) || activities.contains(TPS))
```

Comment:
if you like you could extract above two into defs and put them in BusinessActivities object, then reuse them:
def msbOrTcsp(activities: Set[BusinessActivity]) = activities.contains(MSB) || activities.contains(TrustAndCompanyServices)
  def bpsTdi(activities: Set[BusinessActivity]) = activities.contains(BPS) || activities.contains(TPS)